src/main/java/expression/token/parenthesis/RightParenthesis.java

```diff
@@ -0,0 +1,36 @@
+package expression.token.parenthesis;
+
+import expression.token.TokenType;
+import java.util.Objects;
+
+public class RightParenthesis extends AbstractParenthesis {
+
+  private final String value = ")";
```

Comment:
Might make sense to make such variables `static` when there are going to be many instances of this class. Although default String interning might nullify such concerns